engines/bops_core/spec/jobs/bops_core/auditable_job_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require "bops_core_helper"
+
+RSpec.describe BopsCore::AuditableJob, type: :job do
+  let(:job) do
+    Class.new(ActiveJob::Base) do
+      include BopsCore::AuditableJob
+
+      audit event: "event", scope: "scope", payload: -> { {foo: "bar"} }
```

Comment:
Might alter this call signature to: 

``` ruby
audit "event.scope", -> { {foo: "bar"} }
```

The convention in AS:N is to do things right to left so a typical event would be `status_updated.application.bops_audit` - the last part being added automatically